### ISSUE TITLE
Conversion to RSAKey Error

### ISSUE THREAD

**Issue by emily-crank** [box/box-dotnet-sdk-gen#294]
- [x] I have checked that the [SDK documentation][sdk-docs] doesn't solve my issue.
- [x] I have checked that the [API documentation][api-docs] doesn't solve my issue.
- [x] I have searched the [Box Developer Forums][dev-forums] and my issue isn't already reported (or if it has been reported, I have attached a link to it, for reference).
- [x] I have searched [Issues in this repo][github-repo] and my issue isn't already reported.

### Description of the Issue
Some keys being generated by the Developer tools lead to the error System.Security.Cryptography.CryptographicException: 'The specified RSA parameters are not valid. Exponent and Modulus are required. If D is present, it must have the same length as Modulus. If D is present, P, Q, DP, DQ, and InverseQ are required and must have half the length of Modulus, rounded up, otherwise they must be omitted.'

See https://stackoverflow.com/questions/949727/bouncycastle-rsaprivatekey-to-net-rsaprivatekey for how to create an RSA key from the BouncyCastle objects; the key lengths need to be truncated.

**Comment by emily-crank** [comment]
Hi @arjankowski -

Sorry, I'm working off of credentials provided to me by a customer. I mean the Box developer console. This is related to JWT authentication. 

I've put the correction in fork, located at https://github.com/emily-crank/box-dotnet-sdk-gen/tree/fix/rsa-key. I'll open a PR for you if it helps!

**Comment by arjankowski** [comment]
Hi @emily-crank,
Thanks for the explanation. Because of it, I was able to reproduce it on my end.

This repository is generated by our codegen tool, so we can't directly create a PR with the code to this repository. Instead, we need to add it to our codegen tool. I have already created a PR there based on your fork, so it should appear in this repository soon.

Thanks for pointing out the error and for your help!
Best regards
